frontend/src/css/spinner.css

```
@@ -9,3 +9,28 @@
+    display: block;
+    width: 5rem;
+    max-width: 80px;
+    height: 5rem;
+    max-height: 80px;
+    margin-top: -2.5rem;
+    margin-left: -2.5rem;
+    animation: kiosk--spinner 3.96s cubic-bezier(0.5, 0.65, 0.55, 0.4) infinite;
+    will-change: transform;
+}
+
+@keyframes kiosk--spinner {
```

Stylelint (17.7.0)

[error] Expected keyframe name "kiosk--spinner" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)